src/mdns/mdns_avahi.hpp

```diff
@@ -305,7 +305,7 @@ class PublisherAvahi : public Publisher
 private:
     enum
     {
-        kMaxSizeOfTxtRecord   = 128,
+        kMaxSizeOfTxtRecord   = 256,
```

Comment:
Does this include the length byte?

Reply:
This is actually the `rdata` length of the TXT RR. The maximum should be `65535` but the implementation seems want to avoid too big buffer.

There will be some issues when caller provides longer TXT entries. I think we can refactor to use `stl` later.